include/aws/common/log_channel.h

```diff
@@ -0,0 +1,82 @@
+
+#ifndef AWS_COMMON_LOG_CHANNEL_H
+#define AWS_COMMON_LOG_CHANNEL_H
+
+/*
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Is this really 2010-2019?

Reply:
Well the same question could be asked of pretty much every .c and .h file at this point